### ISSUE TITLE
Uncaught SyntaxError: Unexpected end of input

### ISSUE THREAD

**Issue by endel** [babel/babel-brunch#10]
When you have a JavaScript block without implementation, just with comments, the "unexpected end of input" errors starts to appear.

```javascript
if (true) {
  // this comment breaks everything
}
```

It happens for any statement, even a function:

```javascript
var test = function() {
  // same thing
}
```

**Screenshot:**

![screen shot 2015-04-24 at 4 32 20 pm](https://cloud.githubusercontent.com/assets/130494/7327011/2da23750-eaa0-11e4-85a9-0402e8525361.png)

**Comment by endel** [comment]
Here is the problem in a diff:

**$ diff js_working/app.js js/app.js**

```diff
93a94,95
> "use strict";
>
96,98c98
< var test = function() {
<   // same thing
< }
---
> var test = function test() {};
100c100
< });
---
> // same thing});
```

**Comment by es128** [comment]
Should be fixed in 5.0.1
